code-base/src/main.py

```diff
@@ -55,17 +54,19 @@ def createDirectory(projectName):
         print("[ERROR] A project already exists with the name  '" + projectName + "' ")
 
 
-def basicProject(projectName):
-    if not os.path.exists(projectName):
-        os.makedirs(projectName)
-        os.chdir(projectName)
+def basic_project(project_name):
+    if not os.path.exists(project_name):
+        os.makedirs(project_name)
+        os.chdir(project_name)
         open('.gitignore', 'a').close()
+        os.makedirs('src')
     else:
         print("[ERROR] A project already exists with the name  '" +
-              projectName + "' ")
+              project_name + "' ")
 
-def checkProjType(pType):
-    if pType.lower() in projTypes:
+def check_project_type(project_type):
+    """Check for project types (basic, react, web, etc... here)"""
+    if project_type.lower() in project_types:
```

Comment:
instead of doing project_type.lower() twice in this function I'd just do something like this on line 68:

`project_type = project_type.lower()`
and then just work with that.